### PR TITLE
readme

### PR DESCRIPTION
**Pull request by vicfriedman**
@AnnJohn @jmburges

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
-
+---
+  tags: forms, nested forms, sinatra, intermediate
+  languages: ruby, html
```
Comment (jmburges) [inline README.md]: no longer need this. Put it into the `.learn` file

File: README.md [hunk 2]
```diff
@@ -8,82 +12,14 @@ In this lab, you'll practice using nested forms in Sinatra.
...
-        1
-            name
-                math
-            grade
-                B
-            topic
-                geometry
-```
 
 ## **BONUS**
```
Comment (jmburges) [inline README.md]: Let's drop the bonus. I don't want them going down the AR + Sinatra rabbit hole

**Comment by jmburges** [comment]
few comments and :shipit: 

also close the issues

**Comment by vicfriedman** [comment]
closing

**Pull request closed without merging**